extensions/dacpac/src/wizard/pages/deployPlanPage.ts

```diff
@@ -83,6 +84,9 @@ export class DeployPlanPage extends DacFxConfigPage {
 		this.table.data = [];
 		await this.populateTable();
 		this.loader.loading = false;
+
+		//Reporting DeployDacpac selection event to the telemetry report
+		TelemetryReporter.sendActionEvent(TelemetryViews.DeployPlanPage, 'DataTierApplicationWizardDeployDacpacSelected');
```

Comment:
"DeployDacpacSelected" is a little misleading for this telemetry event. The user already selected deploy dacpac for the previous page. This page is only entered when they are upgrading an existing database